udata/tests/api/test_datasets_api.py

```diff
@@ -1820,7 +1826,7 @@ def test_dataset_schemas_api_list_error_w_cache(self, api, rmock, mocker):
         # Long term cache is used
         response = api.get(url_for('api.schemas'))
         assert200(response)
-        assert response.json == ResourceSchemaMockData.get_expected_v1_result_from_mock_data()
+        assert response.json == ResourceSchemaMockData.get_expected_assignable_schemas_from_mock_data()
```

Comment:
I think we could add a test on filtering out `NON_ASSIGNABLE_SCHEMA_TYPES` in schemas list api?

Reply:
Hum, not sure about what you want. This test already check that the non assignable are not present in the response? And we have a more gobal test here https://github.com/opendatateam/udata/pull/2989/files#diff-36da3294e88ee041b97b82285b97d6d4b365e1835f506171ddab893ecd06b049R774-R778